app/templates/profile.hbs

```diff
@@ -14,11 +14,9 @@
                 <i class="ion-gear-a"></i> Edit Profile Settings
               {{/link-to}}
             {{else}}
-              <button onclick={{perform (if model.following unFollowUser followUser) model}}
-                disabled={{waitingForFollowing}} class="btn btn-sm btn-outline-secondary action-btn"
-                data-test-follow-profile-button>
-                <i class="ion-plus-round"></i> &nbsp; {{if model.following "Unfollow" "Follow"}} {{model.username}}
-              </button>
+              {{#follow-user-button data-test-follow-profile-button disabled=waitingForFollowing class="btn btn-sm btn-outline-secondary action-btn" followAction=(perform (if model.following unFollowUser followUser) model)}}
```

Comment:
I don’t think you need to pass `btn-outline-secondary` class because it will be set when `following` property is false.